src/utilities/index.js

```diff
@@ -19,7 +19,9 @@ export { debounce } from './underscoreMethods';
 export { getModalTitle, MODAL_KEYS } from './getModalTitle';
 export { checkIsObject } from './checkIsObject';
 export { validateReport } from './validateReport';
-
+export { chunk } from './chunk';
 export { parsePositiveIntegerInterfaceInput } from './parsers';
 export { formatErrorItemNames, roundNumber } from './formatters';
 export { MILLISECONDS } from './constants';
+
+export { CHART_CONSTANTS, VACCINE_CONSTANTS } from './modules/vaccines';
```

Comment:
are the chart contstants likely to be used outside of vaccines?

Querying, as the move to expose them here separates them from the `vaccines` file; when importing elsewhere the only context is `utilities/CHART_CONSTANTS` so it's not clear that these might be specific to vaccines.

Reply:
really good point - e36690b4400458447b00442ae477b1dbde676775